packages/auto-drive/package.json

```diff
@@ -34,6 +34,9 @@
   "dependencies": {
     "@ipld/dag-pb": "^4.1.2",
     "blake3": "1.1.0",
+    "blockstore-core": "^5.0.2",
+    "interface-blockstore": "^5.3.1",
```

Comment:
`interface-blockstore` does not seem to be use in your PR, is it a peerDependencies?

Reply:
Thanks for the catch!